src/status_im/subs/wallet/wallet.cljs

```diff
@@ -416,7 +416,8 @@
                                      :total-balance     (utils/calculate-total-token-balance token
                                                                                              chain-ids)))
                             (:tokens account))
-         sorted-tokens (sort-by :name compare tokens)]
+         priority      #(get constants/token-sort-priority (:symbol %) 999)
+         sorted-tokens (sort-by (juxt (comp - :balance) priority) tokens)]
```

Comment:
I think we should extract the `keyfn` function given to `sort-by` and provide it a name, It'll make easier to maintain.

Right now, I'm not sure if it's clear for everyone that this function is returning the pair of numbers:
 `[negative-balance arbitrary-priority]`
 To compare and sort them.
 
@OmarBasem  Could you please refactor this funciton? Since it'll be a pure funciton it can be defined outside the subscription.